h2o-py/h2o/model/model_base.py

```diff
@@ -414,9 +414,24 @@ def varimp(self, use_pandas=False):
         :returns: A list or Pandas DataFrame.
         """
         model = self._model_json["output"]
-        if "variable_importances" in list(model.keys()) and model["variable_importances"]:
-            vals = model["variable_importances"].cell_values
-            header = model["variable_importances"].col_header
+        if self.algo=='glm' or "variable_importances" in list(model.keys()) and model["variable_importances"]:
+            if self.algo=='glm':
+                tempvals = model["standardized_coefficient_magnitudes"].cell_values
+                maxVal = 0
+                sum=0
+                for item in tempvals:
+                    sum=sum+item[1]
+                    if item[1]>maxVal:
+                        maxVal = item[1]
+                vals = []
+                for item in tempvals:
+                    tempT = (item[0], item[1], item[1]/maxVal, item[1]/sum)
+                    vals.append(tempT)
+                header = ["variable", "relative_importance", "scaled_importance", "percentage"]
+            else:
+                vals = model["variable_importances"].cell_values
+                header = model["variable_importances"].col_header
+                
```

Comment:
Here I just grabbed the standardized_coefficient_magnitudes and convert them to the format used for variable importance of other algos.  Now, model.varimp() will provide the same output for all algos including GLM.

Reply:
Perfect, thank you!